### DISCUSSION TITLE
"New" dynamic pages after `npm run export`?

### DISCUSSION THREAD

**Discussion by mledwards** [vercel/next.js#30825] [Help]
Hi there,

Hope someone can help.

I'm exporting my site to use with Capacitor (native app wrapper), and I've hit a stumbling block.

One of my pages is /bookings/[id].js. I understand I can use getStaticProps and getStaticPaths during the export phase to make those pages exist statically, but is there a way of allowing new pages to be created after the export?

The [id] part is related to an id from a datbase, and these would be new bookings made after the export.

**Comment by ahmadnassri** [comment]
@mledwards did you figure this out?
